plugin/al/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,348 @@
+# Migration Guide from AL_USDMaya to ADSK_USDMaya
+
+## Introduction
+
+AL_USDMaya and ADSK_USDMaya[^1] have a lot of similarities [^2]:
+
+- Both proxy shape nodes are inherited from the same base class `MayaUsdProxyShapeBase`
+- Both of the plugins use `vp2RenderDelegate/proxyRenderDelegate` for sub-scene override
+
+In most cases, ADSK_USDMaya could be a drop-in replacement for AL_USDMaya.
+
+Please note that this documentation is not a complete feature comparison of AL_USDMaya and ADSK_USDMaya but mainly focuses on migration from one to another.
+
+[^1] To distinguish Animal Logic's version and Autodesk's version, we call Autodesk's version "ADSK_USDMaya" for convenience.
+
+[^2] This documentation is written based on the current latest release of `maya-usd-v0.27.0`.
+
+## Proxy Shape
+
+### Dependency Nodes
+
+- `AL_usdmaya_ProxyShape`
+
+An equivalent proxy shape type in ADSK_USDMaya is `mayaUsdProxyShape`.
+
+- `AL_usdmaya_LayerManager`
+
+The equivalent node type in ADSK_USDMaya is `mayaUsdLayerManager`.
+
+- `AL_usdmaya_Layer`
+
+Obsolete in AL_USDMaya, no equivalent in ADSK_USDMaya.
+
+- `AL_usdmaya_RendererManager`
+
+Obsolete in AL_USDMaya since moving to VP2 render delegate, no equivalent in ADSK_USDMaya.
+
+- `AL_usdmaya_Scope`, `AL_usdmaya_Transform`, `AL_usdmaya_MeshAnimCreator` and `AL_usdmaya_MeshAnimDeformer`
+
+There is no equivalent nodes like these to [drive transformation and animation from USD to Maya](https://github.com/Autodesk/maya-usd/blob/release/v0.27.0/plugin/al/docs/animation.md) in ADSK_USDMaya, however, the closest implementation is to use [proxyAccessor](https://github.com/Autodesk/maya-usd/blob/release/v0.27.0/lib/mayaUsd/nodes/proxyAccessor.h#L54-L75) for communication between USD and Maya. Find out more usage examples in this [unit test](https://github.com/Autodesk/maya-usd/blob/release/v0.27.0/test/lib/testMayaUsdProxyAccessor.py).
+
+- `AL_usd_ProxyUsdGeomCamera`
+
+This node works similar as MayaUsd ProxyAccessor but for camera, in ADSK_USDMaya, there is native support of USD camera via the [Ufe::Camera interface](https://github.com/Autodesk/maya-usd/blob/release/v0.27.0/lib/usdUfe/ufe/UsdCamera.h#L30), USD camera (and derived schema cameras) could be interacted like other Maya cameras without any problem.
```

Comment:
There are some limitations to the interactions of USD cameras, for instance when viewing through a USD camera you cannot use the panning, zoom, and dolly.

Reply:
Hi @santosd , is there any link to the list of such limitations? e.g. links to https://help.autodesk.com or https://github.com/Autodesk/maya-usd ?

Reply:
Hi @csyshing, The only thing I was able to find was the version where we added visual support for UFE Cameras:
https://help.autodesk.com/view/MAYAUL/2025/ENU/?guid=GUID-985DE87C-E902-49BC-A47D-5699498C4F32
We only added the support to display them in the viewport. The things that are not currently supported with cameras is that when you are looking through a UFE Camera you cannot pan, zoom or tumble. That's pretty much the only known limitations for working with UFE cameras.

Reply:
Added the link in https://github.com/Autodesk/maya-usd/pull/3738/commits/f17d8eea9e48afbea9e3b971badc2d6f4a084381